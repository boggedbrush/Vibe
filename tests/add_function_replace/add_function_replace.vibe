# VibeSpec: 1.2
patch_type: add_function
file: hello.py
--- code: |
    def farewell(name):
        print(f"See you later, {name}!)  # new behavior
