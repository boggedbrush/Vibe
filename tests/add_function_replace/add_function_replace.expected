# hello.py
def greet(name):
    print(f"Hello, {name}!")
def farewell(name):
    print(f"See you later, {name}!)  # new behavior

def farewell(name):
    print(f"Goodbye, {name}!")
